web/modules/custom/portland/modules/portland_location_picker/js/portland_location_picker.js

```diff
@@ -42,18 +42,21 @@
         const SOLVED_ISSUE_MESSAGE = "This issue was recently solved. If that's not the case, or the issue has reoccured, please submit a new report.";
         const ASSET_ONLY_SELECTION_MESSAGE = "We have zoomed in on the address you provided, but this map only allows you to select existing asset markers. There may not be any in the current view. Please search again or use the Locate Me button in the lower-right corner of the map.";
         const VERIFIED_NO_COORDS = "The address you entered is valid, but an error occurred, and it can't be shown on the map. Please zoom in and find the desired location, then click it to set a marker.";
-        const CITY_LIMITS_MESSAGE = "The location you selected is not within the Portland city limits. Please try again."
```

Comment:
Can we keep this message generic? Or at least remove the reference to "report." E.g. TSUP users may be submitting a permit application, not a report.

Reply:
I removed the term "report" but feel the specificity of the rest of the message is helpful.